packages/blade/src/components/Tabs/TabItem.web.tsx

```diff
@@ -128,6 +128,8 @@ const TabItem = ({
     default: 'default',
     hover: 'highlighted',
     focus: 'highlighted',
+    active: 'highlighted',
```

Comment:
Why did we added this active state?  

I changed the `useInteraction` to remove the `active` state since `focus` can just be used as `active` state.

Reply:
Also, 

In this file `packages/blade/src/components/Input/BaseInput/AnimatedBaseInputWrapper.web.tsx`

These if checks will never occur, because while we changed the types the useInteraction hook will never return `active` as a state. It only returns `focus`

<img width="512" alt="image" src="https://github.com/razorpay/blade/assets/35374649/d98c2774-defc-43a5-a725-3460b0162401">

Reply:
Makes sense. I think Input had an active state but we can change it with focus and remove active entirely

Reply:
Done. I've pushed the changes